docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -2737,3 +2762,17 @@ function App(props) {
 
 export default App;
 ```
+
+### TypeError: Cannot read properties of undefined (reading 'pxToRem')
```

Comment:
What's the difference with this case https://deploy-preview-28530--material-ui.netlify.app/guides/migration-v4/#makestyles-typeerror-cannot-read-property-drawer-of-undefined? It seems to be the same root issue, with different symptoms. I guess we won't add a h3 for each instance of where theme.x.y breaks 😁

Reply:
If there are similar issues open, I will group them together. The point that I added here is to help people see the error that match their case (cover as much as possible). I don't have strong evidence to prove my thought but I also don't see a downside of adding it.